src/main/java/org/tikv/common/region/RegionStoreClient.java

```diff
@@ -587,15 +591,15 @@ private boolean isTxnHeartBeatSuccess(TxnHeartBeatResponse resp)
    * @param startTs start version
    * @param commitTs commit version
    */
-  public void commit(BackOffer backOffer, Iterable<ByteString> keys, long startTs, long commitTs)
+  public void commit(BackOffer backOffer, List<ByteString> keys, long startTs, long commitTs)
```

Comment:
Why not convert iterable to stream directly?

Reply:
Or change the type of keys to stream directly to avoid creating redundant concrete List.